Whiteboard/Client/Interfaces/IClientBoardStateManager.cs

```diff
@@ -56,6 +56,7 @@ public interface IClientBoardStateManager
         /// <summary>
         /// Clears the whiteboard to fresh.
         /// </summary>
-        void ClearWhiteBoard();
+        /// <returns>Boolean to freeze whiteboard or not.</returns>
+        bool ClearWhiteBoard();
```

Comment:
Here's the change @bruise-88 .

Reply:
Give me 10 mins I will fix this and launch a PR